frontend/yarn.lock

```diff
@@ -9542,6 +9542,11 @@ send@0.17.1:
     range-parser "~1.2.1"
     statuses "~1.5.0"
 
+serialize-javascript@^1.7.0:
```

Comment:
Could you revert this change? It's duplicated with `2.1.1` below.

Reply:
Or, could you run `yarn upgrade --latest` then just commit the results? I think the approach is ideal. Many old versions will be replaced and dependency problems are resolved.